src/sharedHooks/useSuggestions/useOnlineSuggestions.ts

```diff
@@ -65,9 +75,15 @@ const useOnlineSuggestions = (
     refetch,
     fetchStatus,
     error,
-  } = useAuthenticatedQuery(
+  } = useAuthenticatedQuery<OnlineSuggestionsQueryResponse>(
     queryKey,
-    queryFn,
+    async optsWithAuth => {
+      const params = {
+        ...scoreImageParams,
+        ...( !currentUser && { locale } ),
+      };
+      return scoreImage( params, optsWithAuth ) as Promise<OnlineSuggestionsQueryResponse>;
+    },
```

Comment:
inlining this to infer `optsWithAuth` & fix _this_ type error. We export `ApiOpts` that's technically meant `ApiOpts` and `useAuthenticatedQuery` disagree about null vs undef. I'm just making that not-suggestion's-problem ; D